src/pages/main/search/Search.js

```diff
@@ -0,0 +1,197 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { HiOutlineSearch } from 'react-icons/hi';
+
+const Search = () => {
+  const [value, setValue] = useState('');
+  const [iconActive, setIconActive] = useState(false);
+
+  const isIconActive = () => {
+    setIconActive(true);
+  };
+
+  const hideIconActive = () => {
+    setIconActive(false);
+  };
+
+  return (
+    <SearchWrapper>
+      <InputWrapper>
+        <SearchInput
+          value={value}
+          onChange={e => {
+            setValue(e.target.value);
+          }}
+          onFocus={hideIconActive}
+          onBlur={isIconActive}
+          placeholder="검색어를 입력해 주세요. "
+        />
+        {iconActive && <SearchIcon />}
+      </InputWrapper>
+      <SearchResult>
+        <ResultLeft>
+          <ResultTitle>글 검색 </ResultTitle>
+          <div>
+            {POST_LIST.filter(list => {
+              return list.title.toLowerCase().includes(value.toLowerCase())
+                ? list
+                : null;
+            }).map(list => {
```

Comment:
- false가 null이라면 &&연산자를 사용해도 좋을 것 같습니다.

```suggestion
           {POST_LIST.filter(
              list =>
                list.title.toLowerCase().includes(value.toLowerCase()) && list
            ).map(list => {
```